source/installation/resource-manager/linuxhost.rst

```diff
@@ -0,0 +1,113 @@
+.. _resource-manager-linuxhost:
+
+LinuxHost Adapter
+=================
+
+.. warning::
+
+   This is an experimental feature right now that may change drastically in the
+   future. So always revisit this documentation after upgrading OnDemand.
+
+First Setup cgroups
+********************
+
+.. warning::
+
+   TODO
+
+How To Configure The Adapter
+****************************
+
+A YAML cluster configuration file for a Linux host looks like:
+
+.. code-block:: yaml
+   :emphasize-lines: 10-
+
+   # /etc/ood/config/clusters.d/my_cluster.yml
+   ---
+   v2:
+     metadata:
+       title: "Owens"
+       url: "https://www.osc.edu/supercomputing/computing/owens"
+       hidden: true
+     login:
+       host: "owens.osc.edu"
+     job:
+       adapter: "linux_host"
+       submit_host: "owens.osc.edu"  # This is the head for a login round robin
+       ssh_hosts: # These are the actual login nodes
+         - owens-login01.hpc.osc.edu
+         - owens-login02.hpc.osc.edu
+         - owens-login03.hpc.osc.edu
+       site_timeout: 7200
+       debug: true
+       singularity_bin: /usr/bin/singularity
+       singularity_bindpath: /etc,/media,/mnt,/opt,/run,/srv,/usr,/var,/users
+       singularity_image: /opt/ood/linuxhost_adapter/centos_7.6.sif
+       # Enabling strict host checking may cause the adapter to fail if the user's known_hosts does not have all the roundrobin hosts
+       strict_host_checking: false  
+       tmux_bin: /usr/bin/tmux
```

Comment:
In the cluster config `tmux_bin` or `singularity_bin` could be used to provide a wrapper which cgroups rules could then target.